modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/index/CompanyIndexFactoryTest.java

```diff
@@ -157,26 +138,6 @@ public void testAdditionalTypeMappingsWithRootType() throws Exception {
 		_assertAdditionalTypeMappings();
 	}
 
-	@Test
-	public void testAdditionalTypeMappingsWithRootTypeFromContributor()
```

Comment:
@lipusz FYI that the SPI version is not covered by this one test, but we shouldn't be even needing setting or changing the ' root type' any more so I guess it's acceptable?